docs/rst/maintainer_guide.rst

```diff
@@ -300,6 +300,23 @@ need to update the lock file.
    This command skips the Setuptools and pip-compile update which could disrupt
    your Conda environment.
 
+Continuous integration
```

Comment:
Is this the location of the content, as you imagined it?

Reply:
Yes, this preserves the logical positioning of the anchor.